log/keyvalue.go

```diff
@@ -315,3 +316,8 @@ func Slice(key string, value ...Value) KeyValue {
 func Map(key string, value ...KeyValue) KeyValue {
 	return KeyValue{key, MapValue(value...)}
 }
+
+// Empty returns an KeyValue for an empty value.
```

Comment:
```suggestion
// Empty returns a KeyValue for a empty value.
```

Same comments applies to other factory functions. For me it can be fixed as part of this PR for the sake of consistency.

Reply:
`an` is the correct article for "empty value" given the initial vowel sound when it is pronounced.

```suggestion
// Empty returns a KeyValue for an empty value.
```